gcp/access.py

```diff
@@ -122,8 +122,15 @@ def validate_request(self, access_request_form, request_user, is_group=False):
                 constants.VALID_DOMAIN_REQUIRED_ERROR
             )
 
-        gcp_groups = json.loads(access_request_form.get("selected-gcp-groups"))
         gcp_domain = access_request_form.get("gcp-domain")
+        all_gcp_groups = GCPAccess.get_domain_groups(gcp_domain)
+
+        select_all_groups = json.loads(access_request_form.get("select-all-gcp-groups")) if access_request_form.get("select-all-gcp-groups") else False
```

Comment:
we need to do validation for `select-all-gcp-groups`

Reply:
what is the input type is not at all an array etc.

Reply:
Done.